### ISSUE TITLE
Manage Alert Subscriptions UI issues

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#6685]
- Configurations buttons overlap with text
- Unsubscribe All button is not clearly visible
- No cancel button

![image](https://user-images.githubusercontent.com/20179540/67555110-5057d300-f72e-11e9-9cf0-19f6dc64c16b.png)

Configuration Modal
- Title is small
 - When a configuration is added, the form does not clear

![image](https://user-images.githubusercontent.com/20179540/67555979-fbb55780-f72f-11e9-9742-aa8f059b9058.png)

**Comment by kavishkafernando** [comment]
> No cancel button

@erangatl do we need a cancel button? There are only 3 actions in this UI right?
1. To subscribe to alerts and save
2. Unsubscribe an alert and save
3. Unsubscribe all alerts

IMO I don't think we need a cancel button. @tmkasun @chanaka3d wdyt?

**Comment by erangatl** [comment]
> > No cancel button
> 
> @erangatl do we need a cancel button? There are only 3 actions in this UI right?
> 
> 1. To subscribe to alerts and save
> 2. Unsubscribe an alert and save
> 3. Unsubscribe all alerts
> 
> IMO I don't think we need a cancel button. @tmkasun @chanaka3d wdyt?

IMHO we need one. 
If a user makes some changes, looks around, and later thinks to leave it as it was, there should be a clear emergency exit to that state.
